package.json

```diff
@@ -2,7 +2,7 @@
   "name": "testcafe",
   "description": "Automated browser testing for the modern web development stack.",
   "license": "MIT",
-  "version": "0.23.3-alpha.1",
+  "version": "0.23.3-alpha.2",
```

Comment:
We should update `testcafe-legacy-api` to use `nativeMethods.contentWindowGetter` and `nativeMethods.contentDocumentGetter`.